CHANGELOG-0.8.md

```diff
@@ -0,0 +1,54 @@
+# Changelog since v0.7.0
```

Comment:
Can you also add one more header at the top to make it more obvious what version we're talking about
```
# Release notes for v0.8.0
```

Reply:
That's a new thing, right? We didn't do it like that in older changelogs (e.g. https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-1.6.md).

But I agree, it makes sense. Added.